setup.py

```diff
@@ -20,9 +20,10 @@
 
 
 from setuptools import find_packages, setup
+from marquez_client.version import VERSION
 
 NAME = "marquez-python"
-VERSION = "0.5.6"
+VERSION = VERSION
```

Comment:
Can you make sure that by having the version in a single `version.py` file it still works well with the `release.sh` script.

Reply:
Thanks @roaraya8 . Made the changes for version to be single sourced and updated